testsuite/integration-arquillian/tests/base/src/test/resources/testrealm.json

```diff
@@ -337,6 +432,16 @@
       {
         "client": "third-party",
         "roles": ["customer-user"]
+      },
+      {
+        "client": "test-app-scope",
+        "roles": ["customer-admin-composite-role"]
+      }
+    ],
+    "test-app-scope": [
+      {
+        "client": "test-app-scope",
```

Comment:
It is currently impossible to do this via the frontend. Client Roles for the current client are in the scope automatically. The test-case works correctly but in real-life use is is impossible to map client roles from the current client when not giving it full scope access.